README.md

```diff
@@ -412,15 +675,31 @@ These commands will also print out instructions on how to use the library from M
 
 If you find the version of `simdjson` shipped with `vcpkg` is out-of-date, feel free to report it to `vcpkg` community either by submitting an issue or by creating a PR.
 
+### Usage (Docker)
 
-## Tools
+One can run tests and benchmarks using docker. It especially makes sense under Linux. A privileged access may be needed to get performance counters.
```

Comment:
I think that "A privileged access may be needed to get performance counters." could be reworded as "Privileged access may be needed to get performance counters." (drop the A) I am aware you did not write this sentence.